src/main/java/fr/techad/edc/client/internal/TranslationManagerImpl.java

```diff
@@ -44,7 +45,7 @@ public void loadTranslations(Map<String, Information> publicationInformation) th
                 this.addToDefaultLanguages(entry.getKey(), information);
                 this.addToLanguages(information);
             }
-            this.labels.putAll(this.reader.readLabels(this.languageCodes));
+            this.labels.putAll(this.reader.readLabel(this.languageCodes).getLabels());
```

Comment:
You have to change type of labels.  labels should be I18Content. The goal is we want to define a clear contract. So we want to use the content directly. (Similar to Information, wi use directly the class and not the array for example)